build.props

```diff
@@ -4,11 +4,11 @@
     <EnlistmentRoot>$(MSBuildThisFileDirectory)</EnlistmentRoot>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)\obj</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)obj</BaseIntermediateOutputPath>

```

Comment:
Due to context my point here isn't an issue.
However if this enlistmentroot was being set elsewhere or being relied on in cbt master then we would need to make certain we did something like .trim on \ and then re-add the \ if we want to always ensure that enlistmentroot will always end with a \.